Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12.0.0-alpine as build-deps
+
+RUN apk update && apk upgrade && \
+  apk add --update git && \
+  apk add --update openssh && \
+  apk add --update bash && \
+  apk add --update wget
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+RUN npm install
+
+RUN wget https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh -O /wait-for-it.sh
+RUN chmod +x /wait-for-it.sh
+
+COPY . .
```

Comment:
Again, I think this doesn't relate to the core purpose of the PR and my recommendation is to send it as a separate PR

Reply:
I agree, I've used this locally. But this is not part of the project.